README.md

```diff
@@ -1 +1,200 @@
-# perl-Barriers-PreDefined
\ No newline at end of file
+# NAME
+
```

Comment:
travis status and codecov.io badge can be added here. Please refer to https://github.com/binary-com/perl-Locale-Maketext-Test/blob/master/README.md